src/WattleScript.Interpreter/Extensions.cs

```diff
@@ -28,5 +30,12 @@ public static string GetSourceFragment(this SourceRef[] refs, string fullSourceC
 
             return fullSourceCode.Substring(firstNotNull?.FromCharIndex ?? 0, lastNotNull?.ToCharIndex - firstNotNull?.FromCharIndex ?? 0);
         }
+        
+        public static string GetDescription<T>(this T source)
```

Comment:
I get the idea behind this, but this reflection may break a couple of consumers using Unity IL2CPP (unconditional reflection at runtime), and would be best served with a `ToString().ToLowerInvariant()` as the simplest solution for now

Reply:
fixed in 277b828